task_list.gemspec

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
 
   gem.add_dependency "rack", "~> 1.0"
+  gem.add_dependency "nokogiri", "~> 1.6.0"
```

Comment:
I'd rather not lock Rack/Nokogiri/ActiveSupport if at all possible, otherwise this will block an upgrade to Rails 5, Rack 2, or any future versions of Nokogiri.